src/Form/FormControlLabel.js

```diff
@@ -97,7 +97,7 @@ function FormControlLabel(props: AllProps) {
   const className = classNames(
     classes.root,
     {
-      [classes.hasLabel]: label && label.length,
```

Comment:
I don't really understand this bit actually, shouldn't the class `hasLabel` always be applied ?